adoc/chapters/cuda_backend.adoc

```diff
@@ -0,0 +1,598 @@
+// %%%%%%%%%%%%%%%%%%%%%%%%%%%% begin cuda_backend %%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Putting some general formatting comments here just so the conversation can be threaded:

* The SYCL spec uses `[code]#...#` for code markup not backticks.  All of the backticks should be changed to use `[code]#...#` instead.

* I got the following warnings when building the Asciiidoc sources:
  
  ```
  asciidoctor: WARNING: chapters/cuda_backend.adoc: line 481: id assigned to block already in use: table.extensionsupport
  asciidoctor: WARNING: chapters/cuda_backend.adoc: line 523: id assigned to block already in use: table.extensionsupport
  ```

* The formatting of some of the tables could be improved, but you can only see this in the rendered PDF or HTML version.  I find it easiest to install the Docker container, which allows you to build these yourself.  Another option is to download the "spec-outputs" artifact from GitHub.  (Click on the "Checks" tab in this PR.)

Reply:
This has been dealt with in our internal draft, commit here will be made once other points are dealt with.